docs/DeveloperGuide.adoc

```diff
@@ -341,6 +365,31 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+=== Use case: Delete person
```

Comment:
Perhaps you have forgotten to push the updated commit? It seems like this part is a duplicate of the original template given.